src/applications/personalization/dashboard-2/components/health-care/HealthCareCard.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import NotificationCTA from '../NotificationCTA';
 
-const HealthCareCard = ({ type, cardProperties, noActiveData }) => {
-  let cardTitle = cardProperties?.cardTitle;
-  let line1 = cardProperties?.line1;
-  let line2 = cardProperties?.line2;
-  let line3 = cardProperties?.line3;
-  let sectionTitle = cardProperties?.sectionTitle;
+const HealthCareCard = ({ cardProperties, noActiveData }) => {
+  const cardTitle = cardProperties?.cardTitle;
```

Comment:
We might even be better off removing this reusable component knowing we only use this card once after the design updates.

Reply:
yeah totally. should we make a ticket to do that?

Reply:
I will remove this when I revert the Prescriptions logic, which we already have a ticket for 👍